omod-2.2/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs2_2/ConditionResource2_2.java

```diff
@@ -224,4 +230,30 @@ public String getDisplayString(Condition condition) {
 			return condition.getCondition().getNonCoded();
 		}
 	}
+
+	@Override
+	protected PageableResult doSearch(RequestContext context) {
+		String patientUuid = context.getRequest().getParameter("patientUuid");
+		String includeInactive = context.getRequest().getParameter("includeInactive");
+		ConditionService conditionService = Context.getConditionService();
+		if (patientUuid == null) {
+			return new EmptySearchResult();
+		}
+		Patient patient = ((PatientResource1_8) Context.getService(RestService.class).getResourceBySupportedClass(
+				Patient.class)).getByUniqueId(patientUuid);
+		if (patient == null) {
+			return new EmptySearchResult();
+		}
+		if (includeInactive != null && !includeInactive.isEmpty()) {
+			boolean isIncludeInactive = Boolean.parseBoolean(includeInactive);
```

Comment:
What will happen to the above code when one supplies `yes` or `no`?

Reply:
For all the other inputs other than `true`, `includeInactive` becomes `false`. Should I add a code to handle 'yes/no" like this ?
https://github.com/openmrs/openmrs-module-webservices.rest/blob/4bc9b8a5955d4f373f2784c89d6bccce2db8af27/omod-1.8/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs1_8/ObsResource1_8.java#L449-L458

Reply:
BooleanUtils.toBoolean?

Reply:
I fixed this and a few other issues in the new commit.